grails-app/conf/logback.groovy

```diff
@@ -32,6 +34,7 @@ if (Environment.isDevelopmentMode() && targetDir != null) {
         }
     }
     logger("StackTrace", ERROR, ['FULL_STACKTRACE'], false)
```

Comment:
hm, I am just wondering why do we have log level that high for the dev env.

Reply:
it is the default level, I can decrease it

Reply:
Well, we did not feel the need so far. Please don't bother.